test/api/fallbacks_test.rb

```diff
@@ -22,6 +22,20 @@ def setup
   include I18n::Tests::Localization::Time
   include I18n::Tests::Localization::Procs
 
+  test "fallback does not work with defaults when using en-CH locale" do
+    I18n.with_locale(:'en-CH') do
+      assert_equal 'i exist', I18n.t(:faa)
+      assert_equal 'i exist', I18n.t(:faa, :default => [:'foo.bar'])
```

Comment:
that's the failing test. why should it fail, just because we added a `default` option?

Reply:
Ok, I investigated this a little bit, but I don't have much more time to look into it today.

I suspect this is happening because https://github.com/svenfuchs/i18n/blob/274ed8accd53a573d839617de2dd16b1b01d864b/lib/i18n/backend/base.rb#L33 doesn't take into account the `fallback_in_progress` option. I tried adding `&& !options[:fallback_in_progress]` to this, but it doesn't quite solve it. I am only partially confident that this is the solution.

Could you please take a deeper look at it when you have time, @coorasse?